applets/notification_area/na-grid.c

```diff
@@ -54,7 +66,7 @@ enum
   PROP_ICON_SIZE
 };
 
-G_DEFINE_TYPE (NaGrid, na_grid, GTK_TYPE_BOX)
+G_DEFINE_TYPE (NaGrid, na_grid, GTK_TYPE_GRID)
```

Comment:
This change should be in previous commit.

Reply:
same as above, all the names are grid, but after this first commit the gtkobject is still a box..? SHoudl I still change it even if the applet might probably crash on running?